parse_cli.l

```diff
@@ -494,7 +494,8 @@ gp	{ if (!cmd) {
 					yyless(0);
 					}
 
-<getup2tc>[0-7]			{ up2tc_a[up2tc_idx++] = atoi(yytext);
```

Comment:
What is the difference between these two deltas?  The post-increment operator is deferred per 
   6.5.2.4  Postfix increment and decrement operators

Specifically see statement 2, the value obtained will not have been incremented.

My guess is coverity has a wrong model in this case.